src/000-SCRIPT_OBJ/Helpers.js

```diff
@@ -107,23 +107,24 @@ App.PR = new function() {
     };
 
 	/**
-     * Helper function. Checks relevant statistic config and returns the leveling record if one exists.
-     * @param {string} Type
-     * @param {string} Stat
-     * @param {number} Value
-     * @returns {*}
-     */
-	this.GetLevelingRecord = function(Type, Stat, Value) {
-        var Ratings = this.GetStatConfig(Type)[Stat].LEVELING;
-        var lastSmallerProp;
-        for (var prop in Ratings) {
-            if (!Ratings.hasOwnProperty(prop)) continue;
-            if (prop > Value) break;
-            lastSmallerProp = prop;
-        }
-        if (lastSmallerProp !== undefined) return Ratings[lastSmallerProp];
-        return undefined;
-    };
+	 * Helper function. Checks relevant statistic config and returns the leveling record if one exists.
+	 * @param {string} Type
+	 * @param {string} Stat
+	 * @param {number} Value
+	 * @returns {*}
+	 */
+	this.GetLevelingRecord = function (Type, Stat, Value) {
+	    var Ratings = this.GetStatConfig(Type)[Stat]["LEVELING"];
+	    if (Ratings instanceof Function) return Ratings(Value);
+	    var lastSmallerProp;
+	    for (var prop in Ratings) {
+	        if (!Ratings.hasOwnProperty(prop)) continue;
+	        if (prop > Value) break;
```

Comment:
If the rating is 0 there will be no lastSmallerProp and the function returns undefined, but I'm actually defining 0 in some instances for the leveling - it'll never hit, will it?

Reply:
It is the indent change that makes it look like a new function body. But in fact I just added the line 118. As for the question itself, I think it would be simpler if we use `>=` here. But as far as I remember, all the leveling records are written assuming `>`?